model/task_lifecycle.go

```diff
@@ -1395,6 +1395,9 @@ func UpdateDisplayTaskForTask(t *task.Task) error {
 	if err != nil {
 		return errors.Wrapf(err, "error getting display task for task")
 	}
+	if dt == nil {
+		return errors.Errorf("%s may hold a display task that doesn't exist", t.Id)
```

Comment:
Let's also log an error to splunk with grip.Error so that we can track it on our to see what went wrong. 
Also, I would maybe use that wording for splunk, and use wording like "display task not found for... '%s'"  or something like that.

Reply:
done